Survey/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -24,52 +25,60 @@ struct HomeView: View {
     private let loadUserInfoTrigger = PassthroughSubject<Void, Never>()
     private let loadTrigger = PassthroughSubject<Void, Never>()
     private let willGoToDetail = PassthroughSubject<Void, Never>()
+    private let reloadTrigger = PassthroughSubject<Void, Never>()
     private let minDragTranslationForSwipe: CGFloat = 60.0
 
     var body: some View {
-        LoadingView(
-            isShowing: $output.isLoading,
-            text: .constant(""),
-            content: {
-                ZStack {
-                    setUpTabView()
-                        .overlay(alignment: .top) {
-                            HeaderHomeView(
-                                imageURL: output.user?.avatarUrl ?? "",
-                                isShowingPersonalMenu: $isShowingPersonalMenu
-                            )
-                            .padding(.top, 60.0)
-                            .padding(.leading, 20.0)
-                        }
-                        .overlay {
-                            if isShowingPersonalMenu {
-                                setUpPersonalMenu()
+        RefreshableScrollView { done in
+            reloadTrigger.send()
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+                done()
+            }
+        } progress: { state in
+            RefreshActivityIndicator(isAnimating: state == .loading) {
+                $0.hidesWhenStopped = false
+            }
+            .padding(.top, 30.0)
+        } content: {
+            LoadingView(
+                isShowing: $output.isLoading,
+                text: .constant(""),
+                content: {
+                    ZStack {
+                        setUpTabView()
+                            .overlay(alignment: .top) {
+                                setUpHeaderHomeView()
+                            }
+                            .overlay {
+                                if isShowingPersonalMenu { setUpPersonalMenu() }
                             }
-                        }
+                    }
                 }
+            )
+            .onAppear(perform: {
```

Comment:
I think we should prefer the short terms 
```
.onAppear {
}
```

Reply:
🙏  Fixed at https://github.com/nkhanh44/ic-survey-ios/pull/74/commits/25ef6e3b445b42fd1bf75adba8094ca9b0af366e